src/handler.ts

```diff
@@ -40,57 +40,84 @@ export function getParamsFromUriPath(textDocumentURI: string): Params {
  * @param projects Sentry extension projects configurations.
  * @return Sentry projectID this document reports to.
  */
-export function matchSentryProject(params: Params, projects: SentryProject[]): SentryProject | undefined {
+export function matchSentryProject(
+    params: Params,
+    projects: SentryProject[]
+): { project: SentryProject | undefined; fileMatched: boolean | undefined } {
     if (!projects || !params.repo || !params.file) {
-        return undefined
+        return { project: undefined, fileMatched: undefined }
```

Comment:
What is the difference between `fileMatched: undefined` and `fileMatched: false`?

Reply:
`fileMatched: false` means the file specifically doesn't match the file matching patterns set by the user which will lead to the links not rendering. However if `fileMatched: undefined`, the user did not set a specific pattern for the file and the link will render if there is for example a repo match.